libraries/AP_OSD/AP_OSD_MSP_DisplayPort.cpp

```diff
@@ -77,6 +82,21 @@ void AP_OSD_MSP_DisplayPort::write(uint8_t x, uint8_t y, bool blink, const char
     va_start(ap, fmt);
     int res = hal.util->vsnprintf(buf, sizeof(buf), fmt, ap);
     res = MIN(res, int(sizeof(buf)));
+    const AP_MSP *msp = AP::msp();
+    // enable packed decimal characters only when using ArduPilot's symbol set
```

Comment:
I think I generally see what this is doing - but a link to an explanation or a better one here would be good.

Reply:
@peterbarker  feel free to improve my explanation, not sure I can make it any better